public/views/address.html

```diff
@@ -16,16 +16,29 @@
       </a>
     </div>
   </div>
-  <h1>Address<small data-ng-if="address.balance >= 0">&nbsp;{{address.balance | lisk}} LSK</small></h1>
-  <div class="text-muted" data-ng-if="!(address.balance >= 0)">
-    <span>Loading Address <i class="fa fa-spinner fa-spin"></i></span>
+  <h2>Address:&nbsp;<small>{{address.address}}</small></h2>
+        <div class="text-muted" data-ng-if="!(address.balance >= 0)">
+      <span class="btn-copy" clip-copy="address.address"></span>
+ <span>Loading Address <i class="fa fa-spinner fa-spin"></i></span>
   </div>
   <div data-ng-if="address.balance >= 0">
     <div class="well well-sm ellipsis">
-      <strong>Address</strong>
+  <div class="pull-left">
+        <strong>Address:</strong>
       <span class="text-muted">{{address.address}}</span>
       <span class="btn-copy" clip-copy="address.address"></span>
-    </div>
+        </div>
+      <div class="pull-right hidden-xs hidden-sm">
+        <strong>Public key:</strong>
+        <span class="text-muted">{{address.publicKey}}</span>
+         <span class="btn-copy" clip-copy="address.publicKey"></span>
+        </div>
+         <div class="pkdisplay">
```

Comment:
That not looks so good:
![change](https://cloud.githubusercontent.com/assets/22824146/21464257/3a59dbe8-c978-11e6-932f-4d7a7e00f642.png)

Maybe something like that will be better:
![proposal](https://cloud.githubusercontent.com/assets/22824146/21464259/3da79c72-c978-11e6-89a8-386eb9b97544.png)

But why we need to display public key for mobile at all? For what that can be a useful?